x-pack/plugin/apm-integration/src/internalClusterTest/java/org/elasticsearch/xpack/apm/ApmIT.java

```diff
@@ -160,12 +176,23 @@ public void testSearch() throws Exception {
         final APMTracer.CapturingSpanExporter spanExporter = APMTracer.CAPTURING_SPAN_EXPORTER;
         spanExporter.clear();
 
-        client().prepareSearch()
-            .setQuery(new RangeQueryBuilder("@timestamp").gt("2021-11-01"))
-            .setSearchType(SearchType.QUERY_THEN_FETCH)
-            .setPreFilterShardSize(1)
-            .execute()
-            .actionGet(10, TimeUnit.SECONDS);
+        final Request searchRequest = new Request("GET", "_search");
+        searchRequest.addParameter("search_type", "query_then_fetch");
+        searchRequest.addParameter("pre_filter_shard_size", "1");
+        searchRequest.setJsonEntity("{\"query\":{\"range\":{\"@timestamp\":{\"gt\":\"2021-11-01\"}}}}");
+        searchRequest.setOptions(
+            searchRequest.getOptions()
+                .toBuilder()
+                .addHeader(
+                    "Authorization",
+                    UsernamePasswordToken.basicAuthHeaderValue(
+                        SecuritySettingsSource.TEST_USER_NAME,
+                        new SecureString(SecuritySettingsSourceField.TEST_PASSWORD.toCharArray())
+                    )
+                )
+        );
+
+        final Response searchResponse = getRestClient().performRequest(searchRequest);
```

Comment:
Issue the request at REST layer so it goes through all authentication/authorization layer. Otherwise the first task is created without auth being triggered.